data-raw/parasite_parameters.R

```diff
@@ -0,0 +1,9 @@
+## code to prepare parasite parameters
+parasite_csv <- read.csv("data-raw/parasite_parameters.csv")
```

Comment:
Is this a good spot to add some details about `parasite-parameters.csv`: where is it from, author, date etc?

Reply:
There are some parameter details on the get_parameters function help page, which I think is more accessible to users? I'm happy to add them in here too, if you think that would be good.